### PR TITLE
EI-621 - Enrich ListAllTwinsResponse

### PR DESCRIPTION
**Pull request by conbon**
EI-665 - update protobuf schema, making consistent with search results.

### CODE REVIEW

File: proto/iotics/api/twin.proto [hunk 1]
```diff
@@ -70,7 +87,7 @@ message ListAllTwinsResponse {
 
   // Payload of listed twins.
   message Payload {
-    repeated Twin twins = 1;
```
Comment (mihai-dinculescu) [inline proto/iotics/api/twin.proto]: Is `message Twin {` still used anywhere?

**Review by mihai-dinculescu — APPROVED** [review]
Looks great! Just make sure to announce/plan for the breaking change.

File: proto/iotics/api/twin.proto (hunk 1 above)
Comment (conbon) [reply proto/iotics/api/twin.proto]: Yes, it's used in the DescribeTwinResponse

**Comment by conbon** [comment]
@mihai-dinculescu there is a bit of refactoring going on, but this is just adding fields to the ListAllTwinResponse, it shouldn't be a breaking change.

File: proto/iotics/api/twin.proto (hunk 1 above)
Comment (mihai-dinculescu) [reply proto/iotics/api/twin.proto]: 👍

**Comment by mihai-dinculescu** [comment]
> @mihai-dinculescu there is a bit of refactoring going on, but this is just adding fields to the ListAllTwinResponse, it shouldn't be a breaking change.

There's a chance that the movement of objects in the proto files will cause the deserialization of search results to break if the gRPC clients and the host use different proto files.

**Comment by conbon** [comment]
> > @mihai-dinculescu there is a bit of refactoring going on, but this is just adding fields to the ListAllTwinResponse, it shouldn't be a breaking change.
> 
> There's a chance that the movement of objects in the proto files will cause the deserialization of search results to break if the gRPC clients and the host use different proto files.

Ah ok, I will make sure it's communicated anyway. Thanks

**Comment by StephanieBracaloni** [comment]
Could we catchup about this one. There is a way to go without breaking change. The object refactoring  (sharing the same payload between 2 request/response objects) might be tempting but it is also less flexible. Once again it is a matter of finding the right balance. I think we can share the objects which are describing a resource type with minimal information (ex: Feed, LangString, Property, ...) 
And we can "duplicate" the payload aggregating the "base/type objects".
That way if we want to change the search response payload we can do it without impact on the other services, etc.

Here we should be able to enrich the https://github.com/Iotic-Labs/api/blob/0d3ce55989c742c4bfc236b00143a50d24d54a60/proto/iotics/api/twin.proto#L73
with an inner/nested message that looks like twin + verify grpc compatibility

**Pull request closed without merging**